### ISSUE TITLE
Adjust height of marker lines when plotting

### ISSUE THREAD

**Issue by Sputn1k7** [hyperspy/hyperspy#1747]
I am plotting eds data with a s.sum().plot(True) , which pulls from a list of predefined elements

However, in the interest of making sure small signal peaks get represented properly, I am artificially limiting the ymax of the plot (dividing ymax/5).

This has the unwanted side effect of cutting off some of my elemental labels (specifically the Si_Ka label, as my samples are on Silicon).

A quick search in documentation didn't reveal anything about adjust marker height. Is there any function that would either:

1) Reduce marker height or make it configurable
2) Allow me to increase relative height of minor peaks so they are more obvious in the plot (maybe re-plotting y axis as log?)

**Comment by Sputn1k7** [comment]
partially answered my own question by replotting Y-axis in Log, but my first comment still stands.

**Comment by ericpre** [comment]
Yes, it could be possible to add an argument to the `plot` method to adjust the height of the marker.
